### DISCUSSION TITLE
Copy selection as markdown

### DISCUSSION THREAD

**Discussion by WoodyWoodsta** [outline/outline#6583] [Ideas]
I noticed you can copy an entire document as markdown, but it would be really handy to copy only the selection as markdown! Is this something that has been considered before?

**Comment by tommoor** [comment]
It could be possible with a keyboard shortcut perhaps, would be tricky to discover though

**Reply by WoodyWoodsta** [reply]
I agree, but perhaps it's not something that needs to be commonly used (although I have a use case where I'd end up using it every few days).

What's the appetite for extending the popup toolbar with an extra option?

<img width="679" alt="image" src="https://github.com/outline/outline/assets/8268040/80af8a8d-6989-452b-b173-5c1155b93fb4">

Similar to if you select code:

<img width="264" alt="image" src="https://github.com/outline/outline/assets/8268040/e88369a9-3cda-4a7e-addf-ea98bd9cf301">

Another solution could be a user setting to change the default copy of rich text (I assume the default copy command copies in rich text) to markdown?
